packages/graphql-server-express/src/expressApollo.ts

```diff
@@ -42,33 +42,48 @@ export function graphqlExpress(options: GraphQLOptions | ExpressGraphQLOptionsFu
     }
 
     const formatErrorFn = optionsObject.formatError || graphql.formatError;
+    let buffer;
```

Comment:
can we call this `requestPayload` or something else that's more descriptive than `buffer`?